src/app/Component/Common/Header.css

```diff
@@ -105,3 +105,67 @@
     align-items: center;
     padding-left: 25px;
 }
+
+.header-star-main{
+    width: 300px;
+    height: auto !important;
+    position: absolute;
+    top:10%;
+    right: 10%;
+    background: #50535a;
+    box-shadow: 0px 0px 30px rgb(14 10 16 / 84%);
+    border-radius: 10px;
+}
+.header-star-img{
+    display: flex;
+}
+.header-star-user{
+    margin: 5px;
```

Comment:
need explanation for this change

Reply:
I added a another option called star messages in options.
To satisfy the four options height and width I made changes

Reply:
Why do we need margin: 5px here?

Reply:
i used to display user profile pic and user name in same line. username is not aligned properly thats why i used margin 5 px to align the username and user pic align in same line propely